src/lib/net/classes/methods/get/index.ts

```diff
@@ -1,28 +1,28 @@
 import { EventListener } from "../../../../events";
 import { Users } from "../../../../user";
 import { DefaultUserDeclaration } from "../../../../user/default/types";
-import Client from "../../../utilities/client";
-import Server from "../../../utilities/server";
+import { Client } from "../../../util/client";
+import { Server } from "../../../util/server";
 import { AbstractEndpoint } from "../endpoint/endpoint";
 import { GETDeclaration } from "./types";
 
 export class GetEndpoint<S, R> extends AbstractEndpoint implements GETDeclaration<S, R> {
 	public when(): EventListener<[R]> {
 		const eventListener: EventListener<[R]> = new EventListener();
 
-		Client.listen(this.id, (value: never) => eventListener.call(value));
+		Client.listen(this.id, value => eventListener.call(value));
 
 		return eventListener;
 	}
 
 	public reply(func: (user: DefaultUserDeclaration, value: S) => R): void {
-		Server.listen(this.id, (player: Player, value: never) =>
-			Server.send(this.id, [player], func(Users.get(player), value) as {}),
+		Server.listen(this.id, (player: Player, value) =>
+			Server.send(this.id, [player], func(Users.get(player), value as never) as {}),
 		);
 	}
 
 	public send(toSend: S): void {
-		Client.send(this.id, toSend);
+		Client.send(this.id, toSend as never);
```

Comment:
Is there a way to get rid of this casting?

Reply:
Yes and no, I think I'll have to play with their types.

Reply:
Sorry my brother in christ, but this won't work without having to rewrite the types.